packages/utils/src/jwk-did-utils.ts

```diff
@@ -112,3 +113,112 @@ export function generateJwkFromVerificationMethod(
   const { publicKeyHex, keyType: extractedType } = extractPublicKeyHex(key)
   return createJWK(keyType, publicKeyHex, keyUse)
 }
+
+const isString = (value: any): value is string => typeof value === 'string'
+
+/**
+ * Validate the JOSE header of a JWT
+ * Only support kid with DID URL
+ * @param header
+ * @param payload
+ * Ref: https://www.w3.org/TR/vc-jose-cose/#using-header-params-claims-key-discovery
+ * Ref: https://www.w3.org/TR/vc-jose-cose/#jose-header-parameters-jwt-claims
+ */
+export const validateHeaderJOSE = (header: any, payload: any) => {
+  // Check for 'alg' (Algorithm - Required)
+  if (!isString(header.alg)) {
+    throw new Error('alg must be present and a string')
+  }
+
```

Comment:
Based on this spec: https://www.w3.org/TR/vc-jose-cose/#jwt-format-and-requirements,
I think we should add an if statement to ensure that `header.alg` is not the string `none`.

```ts
// Reject "none" algorithm as per spec
if (header.alg === 'none') {
  throw new Error('alg must not be "none"')
}
```